schemas/camera_ir/dd_camera_ir.xsd

```diff
@@ -141,21 +255,50 @@
 			<xs:documentation>Frame of a camera</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="surface_temperature">
+			<xs:element name="apparent_temperature">
 				<xs:annotation>
-					<xs:documentation>Surface temperature image. First dimension : line index (horizontal axis). Second dimension: column index (vertical axis). The size of this matrix is assumed to be constant over time</xs:documentation>
+					<xs:documentation>Processed image in apparent temperature. First dimension : row index (horizontal orientation). Second dimension: column index (vertical orientation). The size of this matrix is assumed to be constant over time</xs:documentation>
 					<xs:appinfo>
 						<type>dynamic</type>
 						<coordinate1>1...N</coordinate1>
 						<coordinate2>1...N</coordinate2>
 						<units>K</units>
-						<introduced_after_version>3.34.0</introduced_after_version>
 					</xs:appinfo>
 				</xs:annotation>
 				<xs:complexType>
 					<xs:group ref="FLT_2D"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="integration_time">
+				<xs:annotation>
+					<xs:documentation>Integration time per pixel. First dimension : row index (horizontal axis). Second dimension: column index (vertical axis). The size of this matrix is assumed to be constant over time. Allocate dimensions to 1 in case of uniform exposure time.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>Integration time per pixel. First dimension : row index (horizontal orientation). Second dimension: column index (vertical orientation). The size of this matrix is assumed to be constant over time. Allocate dimensions to 1 in case of uniform exposure time.</xs:documentation>
```